### DISCUSSION TITLE
admin.txt

### DISCUSSION THREAD

**Discussion by arond1** [felddy/foundryvtt-docker#525] [Q&A] [answered]
Hello, i'm on debian, I created the container and i can't find the admin.txt file evena fter a find -iname and a find | grep -i does the container rewrite the file ?

**Comment by felddy** [comment, marked as answer]
The `admin.txt` and `options.json` files are created at container startup from the values passed via environment variables and secrets unless `CONTAINER_PRESERVE_CONFIG` is set to `true`.  

See:
- https://github.com/felddy/foundryvtt-docker#configuration-management
- https://github.com/felddy/foundryvtt-docker#using-secrets
- https://github.com/felddy/foundryvtt-docker#optional-variables
